website/docs/d/container_registry.html.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: "digitalocean"
+page_title: "DigitalOcean: digitalocean_container_registry"
+sidebar_current: "docs-do-datasource-container-registry"
+description: |-
+  Get information on a container registry.
+---
+
+# digitalocean_container_registry
+
```

Comment:
```suggestion
**Note**: DigitalOcean Container Registry is currently in private beta. This feature will become available to the general public soon.
```

Reply:
I've notice some providers have callout boxes, eg https://www.terraform.io/docs/providers/google/index.html

Looking at the source, it seems to be some custom markup:

```
-> **Note**: DigitalOcean Container Registry is currently in private beta. This feature will become available to the general public soon.
```

You can preview it with `make website`

Reply:
We can add this to the resource as a warning about being in beta?

```
	// If non-empty, this string is emitted as a warning during Validate.
	DeprecationMessage string
```

Reply:
I added the note @andrewsomething recommended.